src/utils/Utils.ts

```diff
@@ -1587,4 +1588,65 @@ export default class Utils {
   public static hash(data: string): string {
     return crypto.createHash('sha256').update(data).digest('hex');
   }
+
+  public static buildSiteAreaTree(siteAreaId: string, siteAreaList: SiteArea[]): SiteArea {
```

Comment:
Call the renamed buildSiteAreaTree() and rename this one findRootElementInSiteAreaTree() or use a simple find JS method and remove this one.